bpf/types.h

```diff
@@ -69,6 +69,8 @@ typedef __u64 u64;
 #define MAX_OBSERVED_INTERFACES 6
 #define OBSERVED_DIRECTION_BOTH 3
 
+#define MAX_PAYLOAD_SIZE 512
```

Comment:
we may have to document this as a limitation, wdyt?

Reply:
if the packet header exceed 512 bytes we will cap at 512 which should be enough for most of ipv4 and ipv6 basic pkts but we have to place a limit I can't use dynamic size to reserve ringbuf even I will get verifier errors I can make it even larger 4k maybe ?

Reply:
I used 4k to be on the safe side specially with IPv6 and we need to doc this limit

Reply:
Could it be configurable ? 
That would be the best !

Reply:
no it has to be static when u define the structure

Reply:
actually 256 is more than enough even for v6 case v6 base hdrs is just 40 bytes so I will go back to conservative hdr size array unless that caused issues in the future and yes we need to doc this